src/main/java/seedu/duke/template/RcTemplate.java

```diff
@@ -93,6 +95,7 @@ public String toString() {
      */
     @Override
     public LoadComponent getComponent(String component) throws DukeException {
+        assert component.equals("r") || component.equals("c");
```

Comment:
This is already tested in a JUnit test under AddCommand test, hence will cause a build gradle error. Consider removing this.

Reply:
Sure, will remove it.